src/components/Dialogs/ImageCropDialog/ImageCropDialog.tsx

```diff
@@ -17,9 +17,14 @@ import {
 } from './ImageCropDialog.style'
 import { SvgGlyphPan, SvgGlyphZoomIn, SvgGlyphZoomOut } from '@/shared/icons'
 
+export type Dimensions = {
+  width: number
+  height: number
+}
+
 export type ImageCropDialogProps = {
   imageType: CropperImageType
-  onConfirm: (croppedBlob: Blob, croppedUrl: string, imageCropData: ImageCropData) => void
+  onConfirm: (croppedBlob: Blob, croppedUrl: string, imageCropData: Dimensions) => void
```

Comment:
I think we could actually get rid of `cropper.ts`  in `src/types` with the `ImageCropDialog` type.
We should also remove `import { ImageCropData } from '@/types/cropper'` at the top of this file

Reply:
On the second thought, maybe it's a better idea to move `Dimensions` to `@/types/cropper`? wdyt? Also don't like the name Dimensions - it seems to me too broad.

Reply:
You're right, `AssetDimensions` should be better, can't get more specific because this type goes for all assets